src/reminders/Reminder.ts

```diff
@@ -0,0 +1,77 @@
+import type { Moment } from 'moment';
+import { getSettings } from '../Config/Settings';
+
+export class ReminderSettings {
+    notificationTitle: string = 'Task Reminders';
+    dateFormat: string = 'YYYY-MM-DD';
+    dateTimeFormat: string = 'YYYY-MM-DD h:mm a';
+    dailyReminderTime: string = '9:00 am';
+    refreshInterval: number = 5 * 1000; // in Miliseconds
```

Comment:
Suggest renaming the setting to something like `refreshIntervalMilliseconds` (2 Ls in milli)